### PR TITLE
[NEXUS-8011] Select capability type before showing add panel

### PR DESCRIPTION
**Pull request by adreghiciu**
https://issues.sonatype.org/browse/NEXUS-8011
http://screencast.com/t/5C8FPOKnG

### CODE REVIEW

File: plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/controller/Capabilities.js [hunk 1]
```diff
@@ -279,9 +296,9 @@ Ext.define('NX.coreui.controller.Capabilities', {
 
     if (list) {
       me.getCapabilityTypeStore().load(
-        function() {
-          me.reselect();
-        }
+          function() {
```
Comment (kellyrob99) [inline plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/controller/Capabilities.js]: is 4 spaces the desired format? been using 2 spaces in js for so long it looks... strange

**Comment by kellyrob99** [comment]
+1 and tnx @adreghiciu for including the screencast, it really helps

File: plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/controller/Capabilities.js (hunk 1 above)
Comment (adreghiciu) [inline plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/controller/Capabilities.js]: I'm also using 2 spaces. Dunno why IDEA reformat those lines

Comment (adreghiciu) [inline plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/controller/Capabilities.js]: Found it: http://screencast.com/t/2uWRgBkl
And that is in out standard code style https://github.com/sonatype/codestyle/blob/master/sonatype-idea.xml

Comment (kellyrob99) [inline plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/controller/Capabilities.js]: Tnx for the followup

**Comment by djsauble** [comment]
Can we show the first line of the description as a second column in that grid? Scanning a list is much faster than expanding each item individually.

If the description exceeds the available space, can we use a tooltip to show the rest the way Repository Health Check does?

screen shot 2015-02-05 at 11 12 34 am

**Comment by adreghiciu** [comment]
Agree that a tooltip will be better.

**Comment by adreghiciu** [comment]
Updated to use tooltip and show first part of about text. Note that about text is an HTML so I show only the part till first dot (.)

**Comment by djsauble** [comment]
+1. Looks fantastic.

**Comment by jdillon** [comment]
+1... missing > indicator though?

**Comment by djsauble** [comment]
That’s true. I can’t see why they’re missing though.

**Pull request closed without merging**